magic_the_gathering/game_modes/base.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class GameMode:
+    # TODO: Are these attributes sufficient to define a game mode?
```

Comment:
I assume here we are differentiating between a game mode (limited=40 cards in deck, constructed=60 cards in deck most of the time, commander=100 singleton cards in deck, 40 starting life total, commander rules...) and formats (ex: you can play regular constructed in the standard format, modern format, pioneer format, pauper format ... that only change which cards are legal

Reply:
Yes probably, I am a bit confused about the game mode and the formats. I wanted to take inspiration from https://github.com/magefree/mage/blob/d199640fb521e946de41da884e6100b1732b52d5/Mage.Tests/src/test/java/org/mage/test/load/LoadTest.java#LL47C14-L47C14